tests/types/types.go

```diff
@@ -18,6 +18,15 @@ import (
 	"fmt"
 )
 
+const (
+	sectorSize             = 512 // in bytes
+	gptHeaderSize          = 34  // in sectors
+	gptHybridMBRHeaderSize = 63  // in sectors
+
+	IgnitionAlignment = 2048 // 1MB in sectors
+	DefaultAlignment  = 4096 // 2MB in sectors
```

Comment:
Why are the values here 2x the size in the comments?

Reply:
Sectors are 512 bytes (we don't handle 4k sectors, [see related ct issue](https://github.com/coreos/bugs/issues/2111)). 512 * 2048 = 1M